modules/administration-guide/partials/proc_managing-backups-through-custom-resources.adoc

```diff
@@ -30,18 +30,24 @@ metadata:
 spec:
   backupServerConfigRef: backup-server-configuration <1>
 ----
-<1> Name of `CheBackupServerConfiguration` object that defines which backup server to use.
 
-To request a new backup create a new `CheClusterBackup` object.
-To reuse the same name, delete old object first.
+<1> Name of the `CheBackupServerConfiguration` object defining what backup server to use.
+
+* A request to create a new backup starts by creating the`CheClusterBackup` object.
```

Comment:
What do you think about: "Creation of a `CheClusterBackup` object starts a new backup"